tests/cases/fourslash/todoComments20.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: dir1/node_modules/todoTest0.ts
+//// // TODO

```

Comment:
The way the test code reads is entirely unintuitive (not your fault, but the semantics of the test infra). Some clarifying comments in the test would be good for future readers, else looks good to me. Thanks!